.gitignore

```diff
@@ -117,7 +117,7 @@ tools/*.dll
 
 # Sensitive files
 *.keys
-!Azure.AspNetCore.DataProtection.Keys
+!Azure.Extensions.AspNetCore.DataProtection.Keys
```

Comment:
More of a question, but why "AspNetCore"? I mean, it's really subtle, but the same configuration builder is used for Azure Functions, for example. Just to disambiguate?

Reply:
Azure Functions uses AspNetCore.

But you are right that there is a slight difference between Microsoft.Extensions libraries and Microsoft.AspNetCore libraries. @KrzysztofCwalina felt that grouping everything into Azure.Extensions.AspNetCore on our side is cleaner.